activation/activation_test.go

```diff
@@ -98,7 +98,7 @@ func (n *NetMock) hookToAtxPool(transmission []byte) {
 }
 
 func NewMockSigner() *MockSigning {
-	return &MockSigning{signing.NewEdSigner()}
+	return &MockSigning{signing.NewEdSigner([20]byte{})}
```

Comment:
i think this is why using config option was a better approach

Reply:
Made it as an option in https://github.com/spacemeshos/go-spacemesh/pull/3597